lib/src/util/io.dart

```diff
@@ -103,6 +120,36 @@ Stream<List<int>> sanitizeForWindows(Stream<List<int>> input) {
   });
 }
 
+/// Wraps [text] so that it fits within [lineLength], which defaults to
+/// [lineLength].
+///
+/// This preserves existing newlines and doesn't consider terminal color escapes
+/// part of a word's length.
```

Comment:
perhaps call out in the doc comment that words are only split by space and not other whitespace characters

Reply:
Done.